packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 9.0.7
+
+* [java] Adds a `GeneratedApi.FlutterError` exception for passing custom error details (code, message, details).
+* [kotlin] Adds a `FlutterError` exception for passing custom error details (code, message, details).
+* [kotlin] Adds an `errorClassName` option in `KotlinOptions` for custom error class names.
+* [java] Removes legacy try catch from async apis.
+* [java] Removes legacy null check on non-nullable method arguments.
+* Adds `FlutterError` handling integration tests for all platforms.
+
```

Comment:
This list doesn't include the changes you made to the order of items included in a non FlutterError for cpp.